src/lib/media/video/types.ts

```diff
@@ -6,13 +6,16 @@
  */
 
 export interface JobStatus {
```

Comment:
Will we still need this when we ship? Curious if these types are going to be inside the api package.

Reply:
Yeah, it will be. We don't have lexicons for lumi yet